mobile/.bazelrc

```diff
@@ -39,8 +39,7 @@ build --swiftcopt=-wmo
 # https://github.com/bazelbuild/rules_jvm_external/issues/445
 build --repo_env=JAVA_HOME=../bazel_tools/jdk
 build --define disable_known_issue_asserts=true
-# Temporarily set to C++17 because Android NDK does not fully support C++20 yet.
-build:android --cxxopt=-std=c++17 --host_cxxopt=-std=c++17
+build --cxxopt=-std=c++20 --host_cxxopt=-std=c++20
```

Comment:
I think this is duplicated with https://github.com/envoyproxy/envoy/blob/main/.bazelrc#L29?